src/main/cpp/chassis/headingStates/MaintainHeading.cpp

```diff
@@ -39,11 +39,11 @@ void MaintainHeading::UpdateChassisSpeeds(ChassisMovement &chassisMovement)
     {
         chassis->SetStoredHeading(chassis->GetPose().Rotation().Degrees());
         Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, "MaintainDebugging", "Setting Stored Heading (deg)", chassis->GetPose().Rotation().Degrees().to<double>());
-        if (units::angle::degree_t(abs(chassis->GetPose().Rotation().Degrees().to<double>())) < units::angle::degree_t(10.0))
+        if (units::angle::degree_t(abs(chassis->GetPose().Rotation().Degrees().to<double>())) < units::angle::degree_t(15.0))
```

Comment:
This shouldn't be maintain heading this should be a new orientation option

Reply:
This is just helping you get square if you are in maintain, this was updated earlier Nick liked the way it drove